CHANGELOG.md

```diff
@@ -23,6 +23,8 @@
 * DEPENDENCIES
   * Updated DSCParser to version 2.0.0.12.
   * Updated MSCloudLoginAssistant to version 1.1.28.
+* MSFT_IntuneMobileAppConfigurationPolicyIOS
```

Comment:
Please remove the MSFT_ prefix and then place this entry in alphabetical order in the list